src/uberjob/_run.py

```diff
@@ -182,6 +182,7 @@ def run(
     progress: Union[None, bool, Progress, Iterable[Progress]] = True,
     scheduler: Optional[str] = None,
     transform_physical: Optional[Callable[[Plan, Node], Tuple[Plan, Node]]] = None,
+    stale_check_max_workers: Optional[int] = None,
```

Comment:
I wasn't sure exactly what to call this argument, but being able to set it to one when all of the value stores have cached modified times makes a huge difference (>25%).